pumpkin/src/server/mod.rs

```diff
@@ -47,6 +48,8 @@ pub struct Server {
     /// Cache the registry so we don't have to parse it every time a player joins
     pub cached_registry: Vec<Registry>,
 
+    pub open_containers: HashMap<u64, OpenContainer>,
+    pub drag_handler: DragHandler,
```

Comment:
*Luckly*. I refactored the Server struct. Now we split most of the things game related into the world. We should put this into World aswell

Reply:
IMO, containers should not have to be locked on World. World should save a unique ID for each container, and try to lock the container that way.